client/src/containers/App.js

```diff
@@ -34,11 +36,26 @@ class App extends Component {
         });
   }
 
+  openMap() {
+    this.setState({isOpen: true});
+  }
+
+  closeMap(){
+    this.setState({isOpen: false});
+  }
+  
   render() {
     return (
       <div className="container">
         <NavBar />
-        <ApartmentMap markers={this.state.markers} />
+        <HeroCTA
+        openMap={() => this.openMap()}
+        />
+        <ApartmentMap
+          isOpen={this.state.isOpen}
+          markers={this.state.markers}
+          closeMap={() => this.closeMap()}
```

Comment:
Soon these functions will be refactored into redux actions, to provide global access to them, as well as the state of modals, so that App.js doesn't have all that responsibility.